packages/ui/src/common/model/createType.ts

```diff
@@ -95,3 +98,5 @@ const createSafeType = <MTN extends keyof MappedTypeNames, TN extends ValidTypeN
 }
 
 export { createSafeType as createType }
+
+registry.setChainProperties(new GenericChainProperties(registry, CHAIN_PROPERTIES))
```

Comment:
Thanks for the review.

To have have the fake polkadot api operate just like the `ApiRx` all data are re-encoded using `createType` after being send from the web worker to the main thread. But because the metadata is not set on the default registry, the SS58 format is not set either. As a result even thought the account ids object created by `createType('AccountID', ...)` were the same, they where stringyfied to substrate address rather than to the Joystream format. Setting the "Chain properties" manually solves this.

The tests are also failing because of the address format. #4713 matched the mock address format with the QN, but it didn't wasn't apply to the new validator tests.